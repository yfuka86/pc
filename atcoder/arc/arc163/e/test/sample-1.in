5
2
3 1
5
1 1 1 1 1
4
0 0 0 0
4
8 1 6 4
5
3 8 7 12 15
