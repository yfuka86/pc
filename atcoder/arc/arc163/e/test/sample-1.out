Bob
Alice
Bob
Bob
Alice
